
---
bool answer_received
string info
